internal/db/models.go

```diff
@@ -206,6 +206,13 @@ type Deposit struct {
 	UpdatedAt   time.Time `gorm:"not null" json:"updated_at"`
 }
 
+type P2WSHInfo struct {
```

Comment:
Remove this struct, use redeem info as a deposit field

Reply:
done